app/interface_student.py

```diff
@@ -0,0 +1,165 @@
+from aiogram import Bot, types
+from app.states import get_user_state, set_user_state, get_user_data, set_user_data, clear_user_data
```

Comment:
`from states import ...`

Reply:
no, in the app folder there is states.py for states:

ser_states = {}
user_data = {}

def get_user_state(user_id):
    return user_states.get(user_id)

def set_user_state(user_id, state):
    user_states[user_id] = state

def get_user_data(user_id):
    return user_data.get(user_id, {})
...